feat(ai) : adds IA and energy consumption blog post

EFerrandi opened the pull request.

EFerrandi committed:
feat(ai) : adds IA and energy consumption blog post

EFerrandi committed:
docs(ai): add date for CI

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+slug: ia-et-consommation-energetique
+title: IA et consommation énergétique
+authors: [eferrandi]
+date: 2024-06-16
```
```suggestion
date: 2025-06-20
```

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+  - "environnement"
+---
+
+Depuis plusieurs mois nous entendons parler dans les médias de la consommation énergétique inquiétante de l'Intelligence Artificielle (IA). Cette alerte fait désormais parti de notre quotidien, et j'en suis venu à me poser plusieurs questions : quelle est la consommation d'un prompt ChatGPT ? Combien faut-il de prompts pour obtenir l'équivalent d'un plein d'essence ? Ou encore, combien de prompts ...
```
```suggestion
Depuis plusieurs mois nous entendons parler dans les médias de la consommation énergétique inquiétante de l'Intelligence Artificielle (IA). Cette alerte fait désormais partie de notre quotidien, et j'en suis venu à me poser plusieurs questions : quelle est la consommation d'un prompt ChatGPT ? Combien faut-il de prompts pour obtenir l'équivalent d'un plein d'essence ? Ou encore, combien de prompts faut-il pour consommer l'équivalent en euro d'un abonnement Netflix ? 
```
```suggestion
Depuis plusieurs mois nous entendons parler dans les médias de la consommation énergétique inquiétante de l'Intelligence Artificielle (IA). Cette alerte fait désormais parti de notre quotidien, et j'en suis venu à me poser plusieurs questions : quelle est la consommation d'un prompt ChatGPT ? Combien faut-il de prompts pour obtenir l'équivalent d'un plein d'essence ? Ou encore, combien de prompts faut-il pour consommer l'équivalent en euro d'un abonnement Netflix ? 
```

emmanuelperu replied:
Pour l'abonnement Netflix, un euro ça veut tout et rien dire, car tu peux avoir un abonnement sans regarder le même volume que ton voisin. Peut-être mettre "une heure" de Netflix.

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+
+## Contexte et introduction
+
+Avant de commencer à se pencher sur le sujet, il nous faut remettre un peu de contexte. Bien que l'IA existe depuis avant les années 2000, comme le montre l'ordinateur [Deep Blue](https://fr.wikipedia.org/wiki/Deep_Blue) qui a été le premier à vaincre le champion du monde d'échecs en 1997, c'est avec l'apparition des chatbots se basant sur les LLM et ouverts au grand public, tel que ChatGPT, en 2 ...
```
```suggestion
Avant de commencer à se pencher sur le sujet, il nous faut remettre un peu de contexte. Bien que l'IA existe depuis de nombreuses années, comme le montre, par exemple, l'ordinateur [Deep Blue](https://fr.wikipedia.org/wiki/Deep_Blue) qui a été le premier à vaincre le champion du monde d'échecs en 1997, c'est avec l'apparition des chatbots se basant sur les LLM et ouverts au grand public, tel que ChatGPT, en 2022 que l'utilisation de l'IA a explosé. Si avant elle était réservé à des domaines précis et particuliers, aujourd'hui tout le monde peut l'utiliser en quelques clics, que cela soit sur son ordinateur ou son téléphone.
```

emmanuelperu replied:
Elle était réservée.

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+
+Avant de commencer à se pencher sur le sujet, il nous faut remettre un peu de contexte. Bien que l'IA existe depuis avant les années 2000, comme le montre l'ordinateur [Deep Blue](https://fr.wikipedia.org/wiki/Deep_Blue) qui a été le premier à vaincre le champion du monde d'échecs en 1997, c'est avec l'apparition des chatbots se basant sur les LLM et ouverts au grand public, tel que ChatGPT, en 2 ...
+
+C'est cette ouverture au monde qui a poussé une consommation importante de l'IA. Et cette consommation ne s'exprime pas uniquement par le nombre de prompts fait par jour, mais aussi par son aspect énergétique qui se calcule avec l'énergie utilisée par les datacenters, le réseau, les ordinateurs lors des entrainements et bien d'autres paramètres encore. Malheureusement, l'ensemble de ces données s ...
```
```suggestion
C'est cette ouverture au monde qui a poussé une consommation énergétique importante de l'IA. Et cette consommation ne s'exprime pas uniquement par le nombre de prompts faits par jour, mais aussi par l'énergie utilisée par les datacenters, le réseau, les serveurs lors des entrainements et bien d'autres paramètres encore. Malheureusement, l'ensemble de ces données sont difficilement trouvables et calculables. Pourtant nous savons que l'impact est réel, comme l'admet Google dans son [rapport environnemental de 2024](https://blog.google/outreach-initiatives/sustainability/2024-environmental-report/) :
```

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/infographie-usage-massif-intelligence-artificielle.jpg:
Troup lourde et pas en webp.

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/infographie-ia-datacenters-crypto-conso-electrique.jpg:
Troup lourde et pas en webp.

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+
+![Infographie sur la consommation des datacenters pour l'IA](./infographie-ia-datacenters-crypto-conso-electrique.jpg)
+
+Il est compliqué de savoir exactement l'impact total des IA sur l'environnement au travers de sa consommation énergétique. Il y a de nombreux facteurs à prendre en compte et pour cet article, nous avons choisis de nous limiter. Veuillez prendre en compte que les chiffres que nous allons présenter et étudier se base principalement sur la consommation énégétique des prompts. C'est-à-dire sur ce qui ...
```
```suggestion
Il est compliqué de connaître exactement l'impact total de ces IA sur l'environnement au travers de sa consommation énergétique. Il y a de nombreux facteurs à prendre en compte et pour cet article, nous avons choisi de nous limiter. Veuillez prendre en compte que les chiffres que nous allons présenter et étudier se basent principalement sur la consommation énégétique des prompts. C'est-à-dire, sur ce qui est dépensé lorsque nous réalisons une demande à ChatGPT ou une IA similaire. 
```

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+
+La consommation énergétique exacte d'un prompt en IA n'est pas connue. Cependant, selon un [article de ScienceDirect](https://www.sciencedirect.com/science/article/pii/S2542435123003653?dgcid=author), une requête ChatGPT consomme environ 2,9 Wh d’électricité, soit presque dix fois plus qu’une simple recherche Google  estimée à 0,3 Wh. Nous pouvons aussi citer la [Délégation Régionale Académique a ...
+
+Un prompt venant remplacer une simple recherche internet consomme donc 6 à 10 fois l'énergie qui aurait été utilisée si nous l'avions fait à la main. 
```
```suggestion
Un prompt venant remplacer une simple recherche internet consomme donc 6 à 10 fois l'énergie qui aurait été utilisée si vous l'aviez faite à la main sur Google par exemple. 
```

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+
+## Différentes comparaisons de consommations énergétiques
+
+En prenant le temps de chercher sur le web, il est possible de trouver de nombreuses comparaisons entre la consommation des prompts et d'autres outils ou objets fonctionnant à l'électricité, tout comme le montre l'infographie ci-dessous : 
```
```suggestion
En prenant le temps de quelques recherches sur le web (à l'ancienne), il est possible de trouver de nombreuses comparaisons entre la consommation des prompts et d'autres outils ou objets fonctionnant à l'électricité, tout comme le montre l'infographie ci-dessous : 
```

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+## Différentes comparaisons de consommations énergétiques
+
+En prenant le temps de chercher sur le web, il est possible de trouver de nombreuses comparaisons entre la consommation des prompts et d'autres outils ou objets fonctionnant à l'électricité, tout comme le montre l'infographie ci-dessous : 
+![Infographie que l'usage massif de l'IA](./infographie-usage-massif-intelligence-artificielle.jpg)
```
Même si la source est inscrite dans l'image, il faudrait la citer.

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
````diff
@@ -0,0 +1,213 @@
+```
+
+Et voici combien de prompts il faudrait effectué afin d'atteindre les objectifs que nous nous sommes fixés :
+| Modèle | Consomation estimée en Wh | Nombre de recherches Google | Consommer un litre d'essence | Remplir un réservoir de voiture | Un mois d'abonnement à Netflix |
````
Ta voiture de 50 litres pèse 1500 ou 1900 kilos ? Il faut dire familiale moyenne par exemple ;-)

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+
+### Ecrire du code (15000 output tokens)
+
+Pour cet exemple, il aurait été souhaitable d'obtenir un code en 15000 tokens, mais en réalisant la demande aurpès de ChatGTP, voici le résultat qui a été obtenu :
```
```suggestion
Pour cet exemple, il aurait été souhaitable d'obtenir un code en 15 000 tokens, mais en réalisant la demande aurpès de ChatGTP, voici le résultat qui a été obtenu :
```

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+
+La consommation d'un modèle dépend de beaucoup de choses comme son nombre de paramètres d'entrée, et plus la sortie est grande, plus l'IA consomme d'énergie. Comme nous l'avons dit en introduction, utiliser l'IA ne fait qu'augmenter cette consommation. N'oublions pas non plus que nous n'avons pas pris en compte la consommation énergétique des appels sur le réseau ou encore de l'entrainement des I ...
+
+L'objectif de cet article n'est pas de demander à arrêter d'utiliser l'IA, mais à venir nous sensibiliser sur les impacts énergétiques de cette dernière afin que nous puissions l'utiliser plus intelligemment. Arrêtons de tout lui demander dès que nous fassons face à un problème. Si ce dernier peut être résolu avec une simple recherche Google, privilégions là. Servons-nous de l'IA pour les tâches  ...
```
```suggestion
L'objectif de cet article n'est pas de demander à arrêter d'utiliser l'IA, mais à venir nous sensibiliser sur les impacts énergétiques de cette dernière afin que nous puissions l'utiliser plus intelligemment. Arrêtons de tout lui demander dès que nous faisons face à un problème. Si ce dernier peut être résolu avec une simple recherche Google, privilégions là. Servons-nous de l'IA pour les tâches les plus complexes au lieu de nous demander de synthétiser la documentation d'un programme.
```
```suggestion
L'objectif de cet article n'est pas de demander à arrêter d'utiliser l'IA, mais à venir nous sensibiliser sur les impacts énergétiques de cette dernière afin que nous puissions l'utiliser plus intelligemment. Arrêtons de tout lui demander dès que nous faisons face à un problème. Si ce dernier peut être résolu avec une simple recherche Google, privilégions là. Servons-nous de l'IA pour les tâches les plus complexes au lieu de nous demander de synthétiser la documentation d'un programme.
```

emmanuelperu commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,213 @@
+
+## Cela peut aussi vous intéresser
+
+Si vous souhaitez approfondir le sujet ou mettre en place des solutions de votre côté, voici des pistes afin d'aller plus loin :
```
Super cette référence à notre propre blog !!

EFerrandi committed:
docs(ai): updates blog post with webp images and fixes typos

Updates the blog post "IA et consommation energetique" by:

- Replacing the images with `.webp` versions to reduce file size and improve page load times.
- Correcting typographical errors throughout the text to enhance readability.
- Updating the post's date.

EFerrandi committed:
docs(ai): add the equivalent time of video streaming and a new link for exploring more the theme

lefnicolas committed:
fix: typo

mslimani commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,217 @@
+
+## Contexte et introduction
+
+Avant de commencer à se pencher sur le sujet, il nous faut remettre un peu de contexte. Bien que l'IA existe depuis de nombreuses années, comme le montre, par exemple, l'ordinateur [Deep Blue](https://fr.wikipedia.org/wiki/Deep_Blue) qui a été le premier à vaincre le champion du monde d'échecs en 1997, c'est avec l'apparition des chatbots se basant sur les LLM et ouverts au grand public, tel que  ...
```
Le premier ordinateur spécialement développé pour les échecs c'est [Deep Thought](https://fr.wikipedia.org/wiki/Deep_Thought_(ordinateur_d%27%C3%A9checs))

mslimani commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,217 @@
+
+## Consommation énergétique d'un prompt
+
+La consommation énergétique exacte d'un prompt en IA n'est pas connue. Cependant, selon un [article de ScienceDirect](https://www.sciencedirect.com/science/article/pii/S2542435123003653?dgcid=author), une requête ChatGPT consomme environ 2,9 Wh d’électricité, soit presque dix fois plus qu’une simple recherche Google  estimée à 0,3 Wh. Nous pouvons aussi citer la [Délégation Régionale Académique a ...
```
doit-t-on spécifier qu'est-ce qu'un token ?

EFerrandi replied:
Je ne pense pas que cela soit nécessaire, on s'adresse à un public de techs non ? Mais dans le doute je vais ajouter un lien vers une description

mslimani commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,217 @@
+
+La consommation énergétique exacte d'un prompt en IA n'est pas connue. Cependant, selon un [article de ScienceDirect](https://www.sciencedirect.com/science/article/pii/S2542435123003653?dgcid=author), une requête ChatGPT consomme environ 2,9 Wh d’électricité, soit presque dix fois plus qu’une simple recherche Google  estimée à 0,3 Wh. Nous pouvons aussi citer la [Délégation Régionale Académique a ...
+
+Un prompt venant remplacer une simple recherche internet consomme donc 6 à 10 fois l'énergie qui aurait été utilisée si nous l'avions faite à la main sur notre moteur de recherche favori par exemple. 
```
proposition : Un prompt effectuant une recherche internet consomme 6 à 10 fois plus d’énergie qu’une recherche manuelle sur notre moteur de recherche préféré

mslimani commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,217 @@
+En prenant le temps de quelques recherches sur le web (à l'ancienne), il est possible de trouver de nombreuses comparaisons entre la consommation des prompts et d'autres outils ou objets fonctionnant à l'électricité, tout comme le montre l'infographie ci-dessous crée par le [média indépendant Basta!](https://basta.media/) à l'aide des informations regroupées par Hugging Face, l'Université de Carn ...
+![Infographie que l'usage massif de l'IA](./infographie-usage-massif-intelligence-artificielle.webp)
+
+Ces images sont très intéressantes, mais d'un point de vue personnel, cela ne me parle que peu. C'est pour cela que nous allons vous présenter plusieurs cas différents et les comparer à un plein d'essence ou encore au prix d'un abonnement Netflix.
```
proposition :  Ces images sont très intéressantes, mais d’un point de vue personnel, elles ne me parlent que peu. C’est pour cela nous allons vous présenter plusieurs cas différents et les comparer à un plein d’essence ou au prix d’un abonnement Netflix.

mslimani commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,217 @@
+
+Pour l'ensemble de ces tests, nous nous baserons sur les informations suivantes :
+- Une recherche Google vaut 0,3 Wh
+- Nous nous baserons sur les informations fournies par [EcoLogits](https://huggingface.co/spaces/genai-impact/ecologits-calculator)
```
Sur base des informations fournies par

mslimani commented on blog/ai/20250606-IA-et-consommation-energetique/index.md:
```diff
@@ -0,0 +1,217 @@
+| Modèle | Consommation estimée en Wh | Temps équivalent d'une vidéo en streaming | Nombre de recherches Google | Consommer un litre d'essence | Remplir un réservoir de voiture | Un mois d'abonnement à Netflix |
+|-----|-----|-----|-----|-----|-----|-----|
+| ChatGPT 4o | 35,1 | 20,1 min | 117 | 254 | 12 679 | 2 119 |
+| Claude 3 7 Sonnet | 35,1 | 20,1 min | 117 | 254 | 12 679 | 2 119 |
```
Claude 3.7

EFerrandi committed:
fix(ai): handle returns from PR

EFerrandi committed:
fix(ai): handle last returns from PR
